geomstats/geometry/lie_group.py

```diff
@@ -323,9 +315,6 @@ def log_not_from_identity(self, point, base_point, point_type=None):
         -------
         tangent_vec : array-like, shape=[n_samples, {dimension,[n,n]}]
         """
-        if point_type is None:
```

Comment:
how come we don't need this anymore?

Reply:
Is it handled in the decorator?

Reply:
Yes!

Reply:
Right now the decorator does:
```
if point_type is None:
    point_type = self.default_point_type
```
it needs to be generalized to handle the case that we have in invariant_metric, i.e.:
```
if point_type is None:
    point_type = self.group.default_point_type
```

Reply:
Don't worry about it, we decided in #472 that metrics would also have a default_point_type attribute